src/api/app/controllers/application_controller.rb

```diff
@@ -464,10 +464,10 @@ def set_nobody
   end
 
   def set_influxdb_data
-    anonymous = User.current_login == User::NOBODY_LOGIN
+    anonymous = !User.session
 
     InfluxDB::Rails.current.tags = {
-      beta: !anonymous && User.current.in_beta?,
+      beta: !anonymous && User.possibly_nobody.in_beta?,
```

Comment:
why not just `User.possibly_nobody.in_beta?` and getting rid of `anonymous`? :thinking:

Reply:
this was protecting the User.current call - so it could use User.session just as well, but I guess your suggestion makes more sense.